app/helpers/spotlight_helper.rb

```diff
@@ -11,4 +11,13 @@ def display_exhibit_published_date(exhibit)
 
     exhibit.published_at? ? l(exhibit.published_at, format: :long) : 'Unknown'
   end
+
+  def render_markdown_links(options = {})
+    values = options[:value]
+    return ''.html_safe unless values
+    safe_html = values.map do |value|
+      ERB::Util.html_escape(value).gsub(/\[([^\[]+)\]\(([^\)]+)\)/, '<a href="\2">\1</a>')
```

Comment:
Looks good! I could see us wanting to expand available markdown in this field in the future by rendering it the way we seem to for the widgets, i.e. using something like `GitHub::Markup.render('.md', value).html_safe` from Spotlight::RenderingHelper#render_markdown`.